isofit/test/test_lut.py

```diff
@@ -27,29 +27,13 @@
 from isofit.radiative_transfer.modtran import ModtranRT
 
 
-@pytest.fixture(scope="session")
```

Comment:
I discovered this test file and re-enabled it. Now that the LUT branch has been merged, it seems important to actually test it.

Reply:
We still lack a completely operational workflow for topoflux, but agreed that with #441 we're substantially closer, and it's good to have in the test suite.